lib/resources/dstu2/condition.yaml

```diff
@@ -0,0 +1,133 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/condition-definitions.html#Condition
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: Condition
```

Comment:
I know you're following my example in AllergyIntolerance, but these resourceType examples should probably be done like every other example
```
example: '{"resourceType": "Condition"}'
```

I'm not sure why I did AllergyIntolerance that way. I may have been basing it off of something else